doc/build_apps/js_app_tsoa.rst

```diff
@@ -41,10 +41,13 @@ The sample app has the following folder layout:
     │   │   └── site.ts
     │   ├── models
     │   │   └── poll.ts
+    │   ├── services
```

Comment:
Are CCF/enclave apps called services? If not, I would suggest to change services into e.g. enclaveApp. Services is confusing.

Reply:
This term comes from tsoa's docs: https://tsoa-community.github.io/docs/getting-started.html#defining-our-first-model. I've seen this before outside tsoa, another term would be `*Manager` but I'd like to stick to tsoa's docs here. The whole npm app runs inside CCF, not just the services.